JapaneseAddress/Block/Checkout/BillingAddressComponent.php

```diff
@@ -16,11 +18,36 @@
  */
 class BillingAddressComponent implements LayoutProcessorInterface
 {
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfig;
+
+    /**
+     *
+     */
+    const XML_PATH_STORE_LOCALE = 'general/locale/code';
```

Comment:
There is already existing constant for it `XML_PATH_DEFAULT_LOCALE`.

Reply:
removed this line and changed to use Magento\Directory\Helper\Data:: XML_PATH_DEFAULT_LOCALE;